.github/workflows/ci.yml

```diff
@@ -412,6 +413,56 @@ jobs:
             **/surefire-reports/TEST-*.xml
           retention-days: ${{ env.TEST_REPORT_RETENTION_DAYS }}
 
+  test-delta-lake:
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v2
+        with:
+          distribution: 'zulu'
+          java-version: 11
+          cache: 'maven'
+      - name: Configure Problem Matchers
+        run: |
+          echo "::add-matcher::.github/problem-matcher.json"
+      - name: Maven Install
+        run: |
+          export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
+          $RETRY $MAVEN install ${MAVEN_FAST_INSTALL} -am -pl :trino-delta-lake
+      - name: Delta Lake Tests
+        env:
+          ABFS_CONTAINER: ${{ secrets.AZURE_ABFS_CONTAINER }}
+          ABFS_ACCOUNT: ${{ secrets.AZURE_ABFS_ACCOUNT }}
+          ABFS_ACCESS_KEY: ${{ secrets.AZURE_ABFS_ACCESS_KEY }}
+        run: |
+          if [ -n "${ABFS_CONTAINER}" ]; then
```

Comment:
In #10323 that's ready to be merged now we collapse all (notably memsql and bigquery) test jobs into a single one. Can you also do this here to avoid conflicts?

You can add this as an extra step in the `test` job with a condition like this:
```
if: matrix.modules == 'plugin/trino-delta-lake' && env.ABFS_ACCOUNT != ''
```

If all tests in this module require these extra properties to be set, exclude running it in the regular `Maven Tests` step by adding a condition:
```
if: matrix.modules != ':trino-delta-lake'
```

Reply:
I'll prepare a full example.

Reply:
https://github.com/nineinchnick/trino/commit/991bff59decd8494342f51ffea337c6f563d7294

Reply:
copied into this PR. thanks @nineinchnick